x-pack/plugins/synthetics/server/synthetics_service/service_api_client.ts

```diff
@@ -95,6 +95,11 @@ export class ServiceAPIClient {
     return this.callAPI('POST', { ...data, runOnce: true });
   }
 
+  addVersionHeader(req: AxiosRequestConfig) {
+    req.headers = { ...req.headers, 'X-kibana-version': this.kibanaVersion };
```

Comment:
nit: Headers would be lower cased anyways in Node.js as headers are `case-insensitive`, But it would be good to lowercase it in code as well.

Reply:
Got it. IMO it makes sense to make them case-insensitive given the headers will be anyway. I find it really weird that Go's stdlib headers capitalize each word once they parse the all lowercase headers 🤷